### ISSUE TITLE
Some Latin characters cause to_ascii to return an empty result.

### ISSUE THREAD

**Issue by DefiDebauchery** [yamatt/homoglyphs#4]
It's my understanding that STRATEGY_IGNORE should "add characters to result", which to me sounds like it should retain the character in the output if it isn't matched.

However, I cannot seem to retain my complete original input 

```py
import homoglyphs_fork as hgf
hg = hgf.Homoglyphs(strategy=hgf.STRATEGY_IGNORE)

'ß' in hgf.Categories.get_alphabet(['LATIN'])
>>> True

hg.to_ascii('ß')
>>> []
```

This is an issue because there are characters that, while not true homoglyphs, can still be used as them. Consider the German eszett, `ß`, which is a common stand-in for 'B' online.

Because of this, I'm unable to properly detect (as an example) the string 'Сaptchaß𝗈t' -- Cyrillic ES (homoglyph of latin `C`), German Eszett (leet-speak for latin `B`), and Mathematical o (normalized to latin `o`). The best I've been able to achieve is `Captchaot` with strategy LOAD and ascii_strategy REMOVE.

Is there a way to have homoglyphs simply pass-through any character that isn't matched?

**Comment by clydejallorina** [comment]
I believe this is happening as `ß` is not considered as a confusable by the Unicode Consortium. While I believe that this shouldn't be the case, this is also outside the scope of this library as it seems to be basing everything from the [`confusables.txt` file](http://ftp.unicode.org/Public/security/latest/confusables.txt) the Unicode Consortium is providing, which is parsed by `generate.py` to generate the `confusables.json` file that the library then parses for conversion to ASCII.

**Comment by yamatt** [comment]
This has been fixed in v2.1.0

The issue was that the ignore strategy was being applied incorrectly and was effectively the same as the remove strategy.

You will need to now specify the ignore strategy so as not to break existing builds.
```python
import homoglyphs_fork as hgf
hg = hgf.Homoglyphs(ascii_strategy=hgf.STRATEGY_IGNORE)
```
